Tasks/winget/main.ps1

```diff
@@ -327,8 +329,13 @@ else {
     # We're running in package mode:
     if ($Package) {
         Write-Host "Running package install: $($Package)"
+
+        # If there's a version passed, update the version parameter
+        if ($Version -ne '') {
+            $Version = "-Version $Version"
+        }
         
-        $processCreation = Invoke-CimMethod -ClassName Win32_Process -MethodName Create -Arguments @{CommandLine="C:\Program Files\PowerShell\7\pwsh.exe $($mtaFlag) -Command `"Install-WinGetPackage -Id '$($Package)' | ConvertTo-Json -Depth 10 > $($tempOutFile)`""}
+        $processCreation = Invoke-CimMethod -ClassName Win32_Process -MethodName Create -Arguments @{CommandLine="C:\Program Files\PowerShell\7\pwsh.exe $($mtaFlag) -Command `"Install-WinGetPackage -Id '$($Package)' $($Version) | ConvertTo-Json -Depth 10 > $($tempOutFile)`""}
```

Comment:
Single-quote around `$($Version)` to avoid side effects (such as PowerShell commands in the version parameter).

Reply:
Updated, thank you!